app/views/search/_find_form.html.erb

```diff
@@ -4,7 +4,13 @@
       <div class="row">
         <div class="col-12 col-sm-6 col-lg-8">
           <div class="form-group">
-            <%= text_field_tag 'q', @search_term, :id => "search-form", :class => 'form-control', :placeholder => _('Search packages...'), :autocomplete => "off", :autofocus => true %>
+            <div class="input-group mb-3">
+              <div class="input-group-prepend">
+                <button class="input-group-text" type="button" data-toggle="modal" data-target="#search-settings"><%= @baseproject_name %></button>
```

Comment:
With this addition I wonder if we should keep the "settings" button. On one hand, having two buttons for the settings modal is redundant, but on the other hand it is more explicit to have a "settings" button. Maybe someone with more UI/UX expertise can comment on this. (maybe @hellcp?)

Reply:
I would just add spanner icon before name of distro, and remove the settings button. It should be obvious enough.